### DISCUSSION TITLE
ISO 8061 Date Format

### DISCUSSION THREAD

**Discussion by CorruptComputer** [home-assistant/frontend#18689] [Configuration]
I just setup Home Assistant and as I was going through the settings I found this option for date formats.
![image](https://github.com/home-assistant/frontend/assets/5573038/873838e9-6037-429c-8092-ca76cbbeecc0)

I prefer having the date format in the international standard format of [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) (YYYY-MM-DD), however I am disappointed to see that this format is not available in Home Assistant. The closest there is available is YYYY/MM/DD, however I prefer having dashes instead of slashes as it makes it easier to read.

**Comment by Trikos** [comment]
Agree. The standard iso format should be implemented

**Reply by NoRi2909** [reply]
Totally agree! This needs to be fixed. I even consider this to be a bug.

With the YMD option using the language-specific separator this shows up in German as

`2024.10.03`

For any German this is very easy to misread as the 10th of March as we used `DD.MM.YYYY` over here.
By using hyphens this is clearly avoided.

**Reply by ypid** [reply]
> I even consider this to be a bug.

It is not considered a bug according to https://github.com/home-assistant/frontend/issues/21577#issuecomment-2269054397.

**Reply by Tragen** [reply]
But it should be considered a bug. This is crazy.
All date formats are wrong and you cannot select a good one as there is no good one.

**Comment by lindhe** [comment]
Totally agree! Does anyone here know if that functionality is part of the core repo or in the frontend repo?

**Reply by Trikos** [reply]
dunno

**Reply by ypid** [reply]
It is in the frontend according to https://github.com/home-assistant/frontend/pull/16489.

**Reply by lindhe** [reply]
Thanks, @ypid !! That's a lot more helpful than @Trikos reply. 😂 

If I get time, I might look into it.
